### ISSUE TITLE
Default install fails

### ISSUE THREAD

**Issue by iams18i** [shopware/production#72]
After cloning the repo and installing dependencies when running `bin/console system:setup` env variable is missing:

```
In EnvVarProcessor.php line 171:

  Environment variable not found: "APP_URL".
```

**Comment by jkixme** [comment]
+1 Same issue.

**Comment by atipezda** [comment]
+1 same

**Comment by iams18i** [comment]
@jkixme @atipezda  it can be solved by running it this way `APP_URL=http://test.test bin/console system:setup`

**Comment by willopez** [comment]
Experiencing this issue also, it is not present in the `6.1` branch, this seems like a regression on the `6.3` branch.

**Comment by MrSnoozles** [comment]
Still occurring on 6.4 RC 1. That's highly frustrating as it's the first thing new users will encounter when trying to set up shopware.

(As well as most likely choosing the wrong version and using the "development" instead of the "production" repository because  documentation on when to use which is not clear enough)

**Comment by shyim** [comment]
Is fixed in the trunk will be released with 6.4.1.0
